packages/xdl/src/detach/IosNSBundle.ts

```diff
@@ -80,6 +80,8 @@ async function _cleanPropertyListBackupsAsync(
 ): Promise<void> {
   if (context.build?.ios?.buildType !== 'client') {
     await IosPlist.cleanBackupAsync(backupPath, 'EXShell', false);
+    await IosPlist.cleanBackupAsync(backupPath, 'EXSDKVersions', false);
```

Comment:
Do we need this to **only** be run when the `buildType` is not `'client'`? Would it make sense to clear it from clients too?

EDIT: Reread the PR description, I couldn't find `bak` string nowhere in the `ios` directory which would suggest the file is not used, but I'm also ok with not removing it. (Maybe add a comment above?)